owiddata/01.OWID.basicStats.py

```diff
@@ -21,6 +22,7 @@ def convert_date(git_date):
     return datetime.datetime.fromisoformat(git_date).strftime('%B %d, %Y').replace(' 0', ' ')
 
 def billions(count, decimals=0):
+    """Cleans integer in the billions to make a """
```

Comment:
Is part of the comment missing?

Reply:
```suggestion
    """Cleans integer in the billions to make a string formatted as 'X billion', defaulting to integer"""
```

Reply:
Yes! Weird, I thought I typed multiple lines, but clearly something went wrong.